### PR TITLE
Ensure correct phase ID from ANG reader, update point group aliases

### PR DESCRIPTION
**Pull request by hakonanes**
Signed-off-by: Håkon Wiik Ånes <hwaanes@gmail.com>

* Ensure phase IDs are read from ANG file and phases in the header are returned with correct ID
* Add some more point group aliases found in an ANG file, move this dictionary to the `orix.quaternion.symmetry` module
* Separate IO tests into their own files (general IO and ANG, EMsoft and orix plugins)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,3 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - This project now keeps a Changelog
+
+### Fixed
+- Phase IDs found in the ANG file header are used when returning av CrystalMap from this
```
Comment (pc494) [inline CHANGELOG.md:12]: Can you make this a bit shorter? "ANG file reader now recognises Phase IDs defined in the header" perhaps?

File: orix/quaternion/symmetry.py [hunk 2]
```diff
@@ -494,3 +494,11 @@ def get_point_group(space_group_number, proper=False):
         return spacegroup2pointgroup_dict[pgn]["proper"]
     else:
         return spacegroup2pointgroup_dict[pgn]["improper"]
+
+
+# Point group alias mapping
+POINT_GROUP_ALIASES = {
```
Comment (pc494) [inline orix/quaternion/symmetry.py:500]: is there a reason this is all caps? And could we make the comment a bit clearer?

**Review by pc494 — COMMENTED** [review]
Couple of small questions, but broadly looks good to me.

File: orix/tests/io/test_io.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        assert os.path.isfile(temp_file_path) is True
+
+        crystal_map.phases[0].name = "hepp"
+        save(temp_file_path, crystal_map, overwrite=overwrite)
+
+        crystal_map2 = load(temp_file_path)
+        assert crystal_map2.phases[0].name == expected_phase_name
+
+
+def test_loadctf():
```
[the remark this answers is not in the document]
Comment (hakonanes) [reply orix/tests/io/test_io.py:151]: Yep, one you wrote I think?

[the remark this answers is not in the document]
Comment (pc494) [reply orix/tests/io/test_io.py:151]: As it happens it was Ben, it's fairly crude, one to consider deprecating/updating perhaps.... Not action needed for the time being though.

File: orix/quaternion/symmetry.py (hunk 2 above)
Comment (hakonanes) [reply orix/quaternion/symmetry.py:502]: Well, I considered this dictionary a "constant" (https://www.python.org/dev/peps/pep-0008/#constants) at the module level... But I see that other dictionaries in that file are lowercase, will change.

File: CHANGELOG.md (hunk 1 above)
Comment (hakonanes) [reply CHANGELOG.md]: Done.

**Comment by hakonanes** [comment]
By the way, note that I've asked Readthedocs to build our docs (API reference) on every PR. This enables us to check how any changes to the docs will look after merging.

**Comment by hakonanes** [comment]
I wonder if Travis would build quicker if we used .com instead of .org?

**Comment by pc494** [comment]
The .com migration does need to happen soon. Don't think that's what is going on here though, although hard to be sure.

**Comment by pc494** [comment]
Okay, dealing with the .org .com elsewhere, merging this.

**Pull request merged**